Tests/PSKoans.tests.ps1

```diff
@@ -0,0 +1,120 @@
+$here = (Split-Path -Parent $MyInvocation.MyCommand.Path) -replace 'Tests', ''
```

Comment:
I'd suggest a more explicit variable name like `$KoanModulePath` or something along those lines.

Reply:
This follows the "outline" provided by new-fixture in the pester module. If we do end up keeping everything in one module file, I agree it would make sense to rename.